src/server/pfs/server/driver.go

```diff
@@ -580,8 +580,21 @@ func (d *driver) finishCommit(txnCtx *txncontext.TransactionContext, commit *pfs
 	if description != "" {
 		commitInfo.Description = description
 	}
+	// Check that the parent commit is not still open (which can happen in output branches)
+	if commitInfo.ParentCommit != nil {
```

Comment:
Unrelated fix for finishing a commit on an unfinished parent which should never happen.